ibis/expr/api.py

```diff
@@ -1783,7 +1800,8 @@ def _timestamp_strftime(arg, format_str):
     minute=_extract_field('minute', _ops.ExtractMinute),
     second=_extract_field('second', _ops.ExtractSecond),
     millisecond=_extract_field('millisecond', _ops.ExtractMillisecond),
-    truncate=_timestamp_truncate
+    truncate=_timestamp_truncate,
+    time=_timestamp_time,
```

Comment:
This should be modeled as an extract operation, so you should be able to do `_extract_field('time', _ops.ExtractTime)`.